### ISSUE TITLE
Possibly misused work for smoke animation?

### ISSUE THREAD

**Issue by MissPotato** [sp614x/optifine#2520]
When I was trying to find the way to disable smoke particles all together I skipped over Smoke Animation because I thought that was the smoke particle resizing animation and not the whole particle itself. 

Is it intended to be calling Smoke an animation and not a particle?

**Comment by Studio528-real** [comment]
nope, smoke animation turns off smoke (since smoke isnt really a particle it just rises and that is it.), the name is correct.

**Comment by MissPotato** [comment]
But doesn't is still fill the particle table?

**Comment by Studio528-real** [comment]
in 1.14 (maybe even 1.13) particles got their own texture files, do either you can make a resource pack that remove it or you can disable "smoke animation" to get rid of it, this means smoke wont rise from fire and magma under water. I don't see any misuse of a name as disabling "smoke animation" removes smoke, which is what you want right?

**Comment by MissPotato** [comment]
It's not really a big issue just seems inconsistent when compared to other options. For example, particles spawned by the nether portal are called "Portal Particles" in the animations menu.
